test/Conversion/FIRRTLToHW/lower-to-hw-errors.mlir

```diff
@@ -19,4 +19,8 @@ firrtl.circuit "InvalidBundle" {
   func private @UnknownFunction() {
     return
   }
-}
\ No newline at end of file
+  firrtl.module @unprocessedAnnotations() {
+  // expected-warning @+1 {{Unprocessed annotations}}
+    %io_cpu_flush.wire = firrtl.wire {annotations = [{class = "firrtl.transforms.RemainingAnnotation"}]} : !firrtl.uint<1>
+  }
+}
```

Comment:
I would prefer to have this in a separate circuit and to change this `RUN` line to use `-split-input-file`. I expect that this could grow with a lot of test cases.